src/media/css/reviews.styl

```diff
@@ -234,7 +234,14 @@ mkt-prompt[data-modal="flag-review"] {
     }
     a {
         display: block;
+        line-height: 18px;
```

Comment:
Can you use a unitless number here please? When set without a unit then it is a multiple of the `font-size`. I think something like `line-height: 1.5;` or `line-height: 1.25`.